cloud-sql/mysql/mysql/index.js

```diff
@@ -153,14 +94,14 @@ const createPool = async () => {
     }
   }
 
-  if (process.env.DB_HOST) {
-    if (process.env.DB_ROOT_CERT) {
-      return createTcpPoolSslCerts(config);
-    } else {
-      return createTcpPool(config);
-    }
-  } else {
+  if (process.env.INSTANCE_HOST) {
+    // Use a TCP socket when INSTANCE_HOST (e.g., 127.0.0.1) is defined
+    return createTcpPool(config);
+  } else if (process.env.INSTANCE_UNIX_SOCKET) {
```

Comment:
Maybe it's the Go developer in me, but these `else` uses could be removed.

Reply:
As a Python person, I respectfully disagree. 😛 

(I don't feel strongly at all about this, though.)